schematic_db/api_utils/api_utils.py

```diff
@@ -45,12 +58,31 @@ def create_schematic_api_response(
     """
     api_url = getenv("API_URL", "https://schematic.api.sagebionetworks.org/v1/")
     endpoint_url = f"{api_url}/{endpoint_path}"
+    start_time = datetime.now()
```

Comment:
Could you format the time into a certain format? Preferably specifying time zone is US-Pacific?